modules/pricer/src/main/java/com/opengamma/strata/pricer/index/DiscountingOvernightFutureTradePricer.java

```diff
@@ -201,4 +201,15 @@ public PointSensitivities parSpreadSensitivity(ResolvedOvernightFutureTrade trad
     return productPricer.priceSensitivity(trade.getProduct(), ratesProvider);
   }
 
+  /**
+   * Returns the forwardRate rate.
+   *
+   * @param trade  the overnight future trade
```

Comment:
a resolved overnight future trade? Maybe it is worth mentioning that this must be the resolved version

Reply:
Strata doesn't normally refer to the resolved form in Javadoc explicitly